platforms/Windows/installer.wxs

```diff
@@ -2,14 +2,17 @@
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:bal="http://schemas.microsoft.com/wix/BalExtension">
   <Bundle Name="Swift Developer Package for Windows x86_64" Version="$(var.ProductVersion)" Manufacturer="swift.org" UpgradeCode="8c75f32a-7bdf-4c61-abf6-c7e1c4b8fbf6">
     <BootstrapperApplicationRef Id="WixStandardBootstrapperApplication.HyperlinkLicense">
-      <bal:WixStandardBootstrapperApplication LicenseUrl="" LogoFile="Resources/swift.png" SuppressOptionsUI="yes" SuppressRepair="no" />
+      <bal:WixStandardBootstrapperApplication LicenseUrl="" LogoFile="Resources/swift.png" SuppressOptionsUI="no" SuppressRepair="no" />
     </BootstrapperApplicationRef>
 
+    <Variable Name="InstallFolder" Type="string" Value="[WindowsVolume]Library"/>
+
     <Chain>
       <?if $(var.RequiredChain) != "" ?>
       <?foreach MSI in $(var.RequiredChain) ?>
       <MsiPackage SourceFile="$(var.MSI_LOCATION)\$(var.MSI)" Compressed="yes">
         <MsiProperty Name="INSTALL_DEBUGINFO" Value="[INSTALL_DEBUGINFO]" />
+        <MsiProperty Name="INSTALLDIR" Value="[InstallFolder]" />
```

Comment:
Because this unconditionally sets this for all MSIs, does this now not change the default install location for the runtime and ICU?  Both of these have moved to `%ProgramFiles%\swift` in 5.7, so this might actually move them back out?

Reply:
Yeah, I think that we want to start migrating the runtimes to the appropriate places.  See https://github.com/apple/swift-installer-scripts/commit/681954743ec2f83f3d5020c3bece5d53936623d3

This is part of why I think that we might want to hold off on the option being exposed in the installer but allowing exposing it in the MSIs.

Reply:
I see... Looks like we need some advanced path setup, so a user could select different paths for installed features. I've reverted the installer. Now this change is about MSI only.

Reply:
Updated the PR description